package.json

```diff
@@ -16,7 +16,8 @@
     "jade": "~1.11.0",
     "jquery-extendext": "^0.1.2",
     "jquery-ui-bundle": "^1.11.4",
-    "moment": "^2.17.1"
+    "moment": "^2.17.1",
+    "reconcile.js": "github:matthewma7/reconcile.js"
```

Comment:
@matthewma7 should we think about pushing these changes to upstream?

Reply:
Yes. I'll create PR see if the author response at all.
Otherwise, I am thinking we can move this to Kitware organization.